opcua.cpp

```diff
@@ -719,6 +770,95 @@ void OPCUA::clearSubscription()
 	m_subscriptions.clear();
 }
 
+/**
+ * Returns whether Node is to be subscribed to, as per filtering config
```

Comment:
This documentation header should document the parameters and return value.

Reply:
Added